js/app.js

```diff
@@ -40,22 +44,32 @@ titleColourElement.textContent = colour.name;
 document.getElementById("title-character").textContent = playableCharacter;
 document.getElementById("seed").textContent = seed;
 document.getElementById("character").textContent = playableCharacter;
-document.getElementById("starting-items").textContent = getRandomWithSeed(
-  items,
-  seed,
-  3
-)
-  .map(items => `C${items.id} ${items.name}`)
-  .join(", ");
+
+let randomItems = getRandomWithSeed(items, seed, 3, items);
+
+const wiki = new WikiApi();
+
+for (let i in randomItems) {
+  randomItems[i]['thumbnail'] = wiki.getPageInfoResult(randomItems[i]['name']).then((item) => {
+    return '<img src="' + item['thumbnail'] + '" alt="' + randomItems[i]['name'] + '" height="42" width="42">'
+  }).then((item) => {
+    document.getElementById("starting-items-img").innerHTML += item;
+    document.getElementById("starting-items-name").innerHTML = randomItems.map(items => `C${items.id} ${items.name}`).join("<br\> ");
+  });
+}
+
+
+
+
 document.getElementById("end-room").textContent = endingRoom;
-document.getElementById("generate-seed").addEventListener("click", function() {
+document.getElementById("generate-seed").addEventListener("click", function () {
```

Comment:
We should stick to the style already laid out. I presume you have some sort of linter enabled.